downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -82,6 +101,32 @@ public function main() {
 
 ### Improvements
 
+#### Improved resolving logic of constant declarations
+
+- Made the type node in constant declarations optional. Both of the following cases are valid.
+
+    ```ballerina
```

Comment:
Since we don't have bullet points (see previous comment), doesn't have to be indented. Please fix wherever applicable.